app/Models/Search.php

```diff
@@ -175,25 +196,127 @@ private static function decodeSpaces($value) {
 	 */
 	private function parseFeedIds($input) {
 		if (preg_match_all('/\bf:(?P<search>[0-9,]*)/', $input, $matches)) {
-			$ids_lists = $matches['search'];
 			$input = str_replace($matches[0], '', $input);
-			$ids_lists = self::removeEmptyValues($ids_lists);
-			if (!empty($ids_lists[0])) {
-				$this->feed_ids = explode(',', $ids_lists[0]);
-				array_filter($this->feed_ids, function($v) { $v != ''; });
+			$ids_lists = $matches['search'];
+			$this->feed_ids = [];
+			foreach ($ids_lists as $ids_list) {
```

Comment:
Updated to be exact same model as label IDs although it does not make a logical difference for feed IDs because an article cannot be in more than one feed